### PR TITLE
enable ansible-lint gating

### PR DESCRIPTION
**Pull request by srkreddy**
ansible-lint enablement for gating jobs.

### CODE REVIEW

File: gating/check/run [hunk 1]
```diff
@@ -184,7 +184,12 @@ bash scripts/bootstrap-ansible.sh
 
 _download_dependent_components
 
-
+if [ "${RE_JOB_SCENARIO}" = "ansible_lint" ]; then
+   echo "hello ${RE_JOB_SCENARIO}"
+#   find . -type f -regex '.*\.y[a]?ml' -print0 | xargs -0 ansible-lint -x $SKIPLIST --exclude=/etc|| lint_error=1
```
Comment (mvollman) [inline gating/check/run]: Should the above 2 lines be removed now that testing is completed?

Comment (srkreddy) [reply gating/check/run]: updated the commit.

**Comment by srkreddy** [comment]
recheck

**Comment by srkreddy** [comment]
recheck_container_ansible_lint_test

File: gating/check/run [hunk 2]
```diff
@@ -184,7 +184,10 @@ bash scripts/bootstrap-ansible.sh
 
 _download_dependent_components
 
-
+if [ "${RE_JOB_SCENARIO}" = "ansible_lint" ]; then
+   find ./playbooks -type f -regex '.*\.y[a]?ml' -print0 | xargs -0 ansible-lint --exclude=/etc
+   exit
```
Comment (mvollman) [inline gating/check/run]: You need to exit with the return code of ansible lint or this gate will pass every time regardless

Comment (srkreddy) [reply gating/check/run]: Agree, would like to see how the job is behaving first. Will add the return code here in the script, after going through the report and fixing the code.

**Comment by srkreddy** [comment]
recheck_container_ansible_lint_test

**Pull request merged**